lib/ardb/cli.rb

```diff
@@ -106,15 +105,14 @@ def initialize(argv, stdout = nil, stderr = nil)
         @stdout = stdout || $stdout
         @stderr = stderr || $stderr
 
-        @clirb   = Ardb::CLIRB.new
-        @adapter = Ardb::Adapter.new(Ardb.config)
+        @clirb = Ardb::CLIRB.new
       end
 
       def run
         @clirb.parse!(@argv)
         begin
-          Ardb.init
-          @adapter.connect_db
+          Ardb.init(false)
+          Ardb.adapter.connect_db
```

Comment:
@jcredding I'm cool with this in spirit, but we'd need to update the `connect_db` method to actually establish a connection.